Tools/autotest/arducopter.py

```diff
@@ -307,8 +307,13 @@ def fly_throttle_failsafe(mavproxy, mav, side=60, timeout=180):
             # switch back to stabilize
```

Comment:
Interesting comment. I can't figure out why we go to Land mode first - we do it if failsafe fails too.